src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,42 @@
+import Article from "../components/Article";
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { fetchArticles } from "../utils/api";
+import { Grid, Box } from "@mui/material";
+
+export default function ArticlesList() {
+  const [articles, setArticles] = useState([]);
+
```

Comment:
Please add some isLoading functionality so that people can see if the articles are on their way :)